### PR TITLE
Section Setup: req 6, explanations below Manage Students table

### PR DESCRIPTION
**Pull request by Erin007**
[spec](https://docs.google.com/document/d/1YaC8yPL4QUNshj0zXmCZJ1o-FcGYb1Dexv9J0-acHuM/edit?ts=5ea1c1cc)
[LP-1392 ](https://codedotorg.atlassian.net/browse/LP-1392)

Word: 
word

Picture: 
picture

Personal Email: 
email

Clever: 
clever

Google Classroom: 
google

### CODE REVIEW

File: apps/i18n/common/en_us.json [hunk 1]
```diff
@@ -1483,6 +1482,25 @@
...
+  "setUpClass": "Set up your class",
+  "setUpClassCleverIntro": "To get your class set up with Clever, do the following:",
+  "setUpClassClever1": "1. Make sure your class is set up the way you want it to be in Clever.",
+  "setUpClassClever2": "2. Hit the 'sync classroom' button above to copy your Clever students to Code.org.",
+  "setUpClassCleverFinished": "You're finished! If you need to add or remove students later, do that in Clever first, and then sync your classroom again with Code.org.",
+  "setUpClassGoogleIntro": "To get your class set up with Google Classroom, do the following:",
+  "setUpClassGoogle1": "1. Make sure your class is set up the way you it to be in Google Classroom.",
+  "setUpClassGoogle2": "2. Hit the 'sync classroom' button above to copy your Google Classroom students to Code.org.",
+  "setUpClassGoogleFinished": "You're finished! If you need to add or remove students later, do that in Google Classroom first, and then sync your classroom again with Code.org.",
+  "setUpClassPicIntro": "To get your class set with picture password accounts, do the following:",
```
Comment (clareconstantine) [inline apps/i18n/common/en_us.json]: nit: missing "up" (set up with)

File: apps/i18n/common/en_us.json [hunk 2]
```diff
@@ -1538,6 +1556,27 @@
   "signinForProgress": "Already have a Code.org account? Sign in so we can save your progress.",
   "signinOrAge": "Sign in or provide your age to continue",
   "signinDanceParty": "Provide your age to continue",
+  "signingInClever": "Signing in with Clever",
+  "signingInCleverIntro": "Have your students do the following to sign in with thier Clever accounts:",
```
Comment (clareconstantine) [inline apps/i18n/common/en_us.json]: typo: their

File: apps/i18n/common/en_us.json [hunk 3]
```diff
@@ -1538,6 +1556,27 @@
...
+  "signingInClever1": "1. Have your students log in to their Clever account:",
+  "signingInClever1a": "a. Go to [www.clever.com](https://www.clever.com).",
+  "signingInClever1b": "b. Click on the Code.org logo on the Clever dashboard. The logo looks like this:",
+  "signingInGoogle": "Signing in with Google Classroom",
+  "signingInGoogleIntro": "Have your students do the following to sign in with thier Google Classroom accounts:",
```
Comment (clareconstantine) [inline apps/i18n/common/en_us.json]: typo: their

File: apps/i18n/common/en_us.json [hunk 4]
```diff
@@ -1538,6 +1556,27 @@
...
+  "signingInEmail": "Signing in with Personal Logins",
+  "signingInEmailIntro": "Have your students do the following to sign in with their personal logins:",
+  "signingInEmailGoogle1": "1. Go to [www.code.org]({codeOrgLink}) and click the 'sign in' button.",
+  "signingInEmail2": "2. Sign in using the email and password they created.",
+  "signingInPic": "Signing in with Picture passwords",
+  "signingInPicIntro": "Have your students do the following to sign in with their picture passwords:",
+  "signingInPic3": "3. Click on their picture password.",
+  "signingInWord": "Signing in with Secret Word passwords",
+  "signingInWordIntro": "Have your students do the following to sign in with their secret words:",
+  "signingInWordPic1": "1. Go to [{joinLink}]({joinLink}) OR go to [www.code.org]({codeOrgLink}), click the 'sign in ' button and enter the section code {sectionCode} in the section box.",
```
Comment (clareconstantine) [inline apps/i18n/common/en_us.json]: nit: extra space after 'sign in' in the quotes

**Review by clareconstantine — APPROVED** [review]
So many strings! Found a couple teeny string typos but looks good otherwise!

**Comment by Erin007** [comment]
thanks for the careful proofreading @clareconstantine! The number of times I typed "sing in" instead of "sign in" and had to catch myself is embarrassingly high 🙃

**Pull request merged**